### PR TITLE
4408 create rnr list view (PT 1)

### PR DESCRIPTION
**Pull request by lache-melvin**
Start of #4408

# 👩🏻‍💻 What does this PR do?

This is just the frontend, rudimentary list view... PR was getting big enough 😅 

Screenshot 2024-07-16 at 12 10 42 PM

## 💌 Any notes for the reviewer?

Backend incoming!

[This](c2da171b900b76f6d637332b67be31ec024ca083) commit is just moving immunisation programs components to the programs folder.

[This diff](https://github.com/msupply-foundation/open-msupply/compare/c2da171b900b76f6d637332b67be31ec024ca083...a402ce8fdb41259e26a7f90cfb068a53afa55998) adds the new view

# 🧪 Testing

- [ ] Ensure you have the program module enabled
- [ ] In App drawer, you can see the `Programs > R&R forms`
- [ ] Clicking it, you are navigated to R&R forms list view, with a dummy row

# 📃 Documentation

- [x] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

File: client/packages/common/src/intl/locales/en/programs.json
Comment (jmbrunskill) [inline client/packages/common/src/intl/locales/en/programs.json:14]:
```suggestion
  "error.no-rnr-forms": "No R&R Forms found",
```
Not sure if it matters, but thinking we want consistncy?

File: client/packages/host/src/components/Navigation/ProgramsNav.tsx
Comment (jmbrunskill) [inline client/packages/host/src/components/Navigation/ProgramsNav.tsx:23]: Does everyone with the omProgramModule get R&R Forms?
Will we add an extra pref or something?

**Review by jmbrunskill — APPROVED** [review]
Looks good to me, but haven't tested the branch.

You might want a review from Mark or a bit of a test from someone else before merging.

Thought I'd take a quick look to see what's happening with R&Rs...

File: client/packages/common/src/intl/locales/en/programs.json
Comment (lache-melvin) [reply client/packages/common/src/intl/locales/en/programs.json]: for sure thank you!

File: client/packages/host/src/components/Navigation/ProgramsNav.tsx
Comment (lache-melvin) [reply client/packages/host/src/components/Navigation/ProgramsNav.tsx:23]: Chatted with Mark earlier, we thought just program module for now? Could always hide further if needed (though we seem to have the pattern of "show people what they could be using")

**Comment by lache-melvin** [comment]
> You might want a review from Mark or a bit of a test from someone else before merging.

Yeah cool thanks, I'll get him to take it for a spin once the backend is connected 👍

**Pull request merged**
